### ISSUE TITLE
Pytorch Dataloader takes very very long time to load one batch on GPU (it is still slow but much faster when on cpu)

### ISSUE THREAD

**Issue by op-real** [googlecolab/colabtools#1970]
Bug report for Colab: http://colab.research.google.com/.

For questions about colab usage, please use [stackoverflow](https://stackoverflow.com/questions/tagged/google-colaboratory/).

- Describe the current behavior:
Pytorch Dataloader takes very very long time to load one batch on GPU (it is still slow but much faster when on cpu)
- Describe the expected behavior:
Fast
- The web browser you are using (Chrome, Firefox, Safari, etc.):
Chrome
- Link (not screenshot!) to a *minimal, public, self-contained* notebook that
  reproduces this issue (click the *Share* button, then *Get Shareable Link*):
The dataset contains around 4000 images with 178*218 resolution. 

`
class Crop(object):
    def __call__(self, img):
      return F_t.crop(img, 0, 0, 115, 218)

transformer = transforms.Compose([
    transforms.ToTensor(),
    Crop(),
    transforms.Resize(32),
    transforms.RandomHorizontalFlip(),
    transforms.Normalize([0.5,0.5,0.5],[0.5,0.5,0.5]),
])

train_loader = DataLoader(
    train_set,
    batch_size = batch_size,
    shuffle = True,
    pin_memory = pin_memory
)

for images, labels in train_loader :
    # not loading
`

update:
(The problem got resolved by mounting to a different google drive.  Still confusing since the google drive used had a lot of spaces.)

update again:
the problem came back again. Apparently google drive was not the issue. 

update:
the issue is gone without making any changes. Guess something is wrong in the backend.

update:
the issue kept coming back quite randomly.

**Comment by op-real** [comment]
![image](https://user-images.githubusercontent.com/7625857/115948051-bba47180-a499-11eb-9f6c-cdf6fe12da7d.png)
stuck with this

**Comment by cperry-goog** [comment]
I don't think this is a colab issue
